build.gradle

```diff
@@ -586,15 +586,52 @@ project('test:system') {
     }
 
     task execShellScript(type: Exec) {
-        commandLine "./preTestScript.sh"
-        args "$CLUSTER_NAME", "$MASTER", "$NUM_SLAVES"
+
+        ext.awsExecution = project.hasProperty('awsExecution') ? project.awsExecution : 'false'
+
+        if(ext.getProperty('awsExecution').toString().equals("true")) {
+            commandLine './aws/preTestScript.sh'
```

Comment:
Don't use relative path.

Reply:
The working directory in this project `test:system` is `$rootDir/test/system` . I am unable to set two different working directories one for if and other for else. So using this.